dev/ts/main.ts

```diff
@@ -164,7 +166,10 @@ function loadBpmnFromUrl(url: string, statusFetchKoNotifier: (errorMsg: string)
     })
     .then(bpmn => {
       loadBpmn(bpmn);
-      log(`Bpmn loaded from url ${url}`);
+      log(`BPMN content loaded from url ${url}`);
+    })
+    .then(() => {
+      updateStyleOfElementsIfRequested();
```

Comment:
**thought:** the style is applied only when the diagram is loaded by retrieving the content from a URL.
It isn't apply when using the load button or DnD.
This is OK for now: we only need this feature for e2e tests, the feature is not public. We generalize it later.